src/pages/feedback/FeedbackPage.tsx

```diff
@@ -12,50 +12,53 @@ import {
 import { Carousel } from '@mantine/carousel';
 import Autoplay from 'embla-carousel-autoplay';
 import { FormFeedback } from '../../components/feedback/FormFeedback';
-
-const reviews = [
-  {
-    pseudo: 'John Doe',
-    date: '2023-02-01',
-    comment: 'La plateforme est très intuitive, je suis très satisfait !',
-    rating: 1,
-  },
-  {
-    pseudo: 'Jane Doe',
-    date: '2023-01-15',
-    comment:
-      'J’ai eu un problème avec ma réservation, mais le service client a été très réactif et m’a aidé à trouver une solution.',
-    rating: 1,
-  },
-  {
-    pseudo: 'Bob Smith',
-    date: '2022-12-20',
-    comment:
-      'Je suis un peu déçu par la qualité des services proposés, mais le rapport qualité-prix est correct.',
-    rating: 3,
-  },
-  {
-    pseudo: 'Alice Smith',
-    date: '2022-12-01',
-    comment:
-      'Je recommande vivement cette plateforme, j’ai passé un excellent séjour !',
-    rating: 5,
-  },
-  {
-    pseudo: 'Eva Brown',
-    date: '2022-11-15',
-    comment:
-      'Je suis très satisfaite de ma réservation, je n’hésiterai pas à revenir !',
-    rating: 5,
-  },
-];
-
-const averageRating =
-  reviews.reduce((acc, review) => acc + review.rating, 0) / reviews.length;
+import { AnoTestimonialsController } from '../../services/BaseApi';
+import { useMutation } from '@tanstack/react-query';
 
 const FeedbackPage = () => {
   const autoplay = useRef(Autoplay({ delay: 5000 }));
 
+  const testimonialsMutation = useMutation({
+    mutationFn: async () => AnoTestimonialsController.getAllTestimonialGET(),
+  });
```

Comment:
Don't use mutation for get [React Query v5](https://tanstack.com/query/latest/docs/framework/react/quick-start)